README.md

```diff
@@ -6,6 +6,13 @@
 npm install -g autorest
 ```
 
+# Compatability
+This AutoRest extension generates TypeScript code that is compatible with:
+```
+"ms-rest-azure-js": "^0.4.60",
```

Comment:
Consider using ~ versions

Reply:
Done in https://github.com/Azure/autorest.typescript/pull/146/commits/b3d4a6cdc10f12c384032cd7118ca5cca24050c3.